pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -94,16 +99,39 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 	return nil, nil
 }
 
-func callEndpoint(invocationCtx workflow.InvocationContext, input workflow.Data, url string) error {
+func normalizeInput(input []byte) []byte {
+	var scanDoneEvent json_schemas.ScanDoneEvent
+	err := json.Unmarshal(input, &scanDoneEvent)
+	if err != nil {
+		return input
+	}
+
+	durationMs, err := strconv.ParseInt(scanDoneEvent.Data.Attributes.DurationMs, 10, 64)
```

Comment:
Suggestion: You might want to use something like `durationMs float64 `json:",string"`` from https://pkg.go.dev/encoding/json#Marshal. This makes the code much smaller.

Reply:
`int64` would be conventional here but yes, we shouldn't be re-parsing if we can help it.